src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -5765,6 +5767,197 @@ static void M()
 
         #endregion
 
+        #region Patterns
+
+        public void SyntaxOffset_Parrern()
+        {
+            var source = @"class C { int F(object o) { if (o is 33) { return 44; }  if (o is int i) { return i; } if (o is bool) { return 1; } return 0; }}";
```

Comment:
You can also simplify the code like so: 
```
bool F(object o) => o is int i && o is 3 && o is bool;
```

---
In reply to: [113005390](https://github.com/dotnet/roslyn/pull/18883#discussion_r113005390) [](ancestors = 113005390)